java/src/main/java/com/ibm/plugin/rules/detection/bc/BouncyCastleInfoMap.java

```diff
@@ -143,6 +147,7 @@ public String getDisplayName(@Nonnull String key) {
      * @param removePart - Substring to remove from the name (only when no specific name was defined
      *     with {@code putName}). If {@code null}, it does nothing.
      * @return The correct name of the cryptographic asset
+     * @deprecated use the key of the map instead
```

Comment:
Add the `@Deprected` annotation similar to [here](https://github.com/IBM/sonar-cryptography/blob/main/engine/src/main/java/com/ibm/engine/model/context/CipherContext.java#L61-L70)